pages/runtime/supervisor-api.md

```diff
@@ -459,8 +459,8 @@ Returns the application running on the device
 The app is a JSON object that contains the following:
 * `appId`: The id of the app as per the Resin API.
 * `commit`: Application commit that is running.
-* `imageId`: The Docker image of the current application build.
-* `containerId`: ID of the Docker container of the running app.
+* `imageId`: The docker image of the current application build.
```

Comment:
Why lowercase docker? Is it written as code?

Reply:
(same applies for a bunch of lowercase Docker instances below - if it's not written as a command or code, it needs to be uppercase)

Reply:
per Docker: 
>Capitalize the D in Docker, Dockerfile, Docker Swarm and other marks consisting of "Docker", except when you are specifically referring to the commandline command, for example for 'docker run'. Then you may use 'docker'.

Reply:
Another one that is pulled in external. I'll make a PR to get these all fixed.